.github/workflows/release-please.yaml

```
@@ -0,0 +1,47 @@
+name: Release Please
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  track-release:
+    runs-on: ubuntu-latest
+    outputs:
+      release_created: ${{ steps.track-release.outputs.release_created }}
+    steps:
+      - uses:
+          googleapis/release-please-action@a02a34c4d625f9be7cb89156071d8567266a2445 # v4
+        id: track-release
+        with:
+          manifest-file: .github/.release-please-manifest.json
+          config-file: .github/release-please-config.json
+  goreleaser:
+    needs:
+      - track-release
+    if: ${{ needs.track-release.outputs.release_created }}
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          fetch-tags: true
+
+      - name: Set up Go
+        uses: actions/setup-go@0aaccfd150d50ccaeb58ebd88d36e91967a5f35b # v5
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@9c156ee8a17a598857849441385a2041ef570552 # v6
```

I feel like all these come from another repository. But I assume it's from a repository that uses hardened version, aka pinned version.

fuego doesn't pin versions. Maybe it should, but let's keep it out of this PR considerations.

I'm unsure how dependabot will handle them while there are no hardened action.

I might be wrong.

I would use the regular way here


```suggestion
        uses: actions/checkout@v4
        with:
          fetch-tags: true

      - name: Set up Go
        uses: actions/setup-go@v5

      - name: Run GoReleaser
        uses: goreleaser/goreleaser-action@v6
```

This comment is valid for all workflow files that are updated by this PR

---

Your project, your rules!
I undertsand the pinning as a _best practice_ in GHA, but I'm fine with deferring that.

When it comes to dependency management, if dependabot can't handle it, Renovate has a free tier that definitely knows how to handle the version pinning and it's extremely simple to start going with Renovate.

---

Absolutely! :slightly_smiling_face: 

Sorry if it seemed resentful. Not intended at all. :pray:

---

I didn't see this last message on pinning. I'm fine either way but maybe make that pinning in a different PR.